Source/WebCore/platform/ContextMenuItem.cpp

```diff
@@ -265,6 +265,10 @@ bool isValidContextMenuAction(ContextMenuAction action)
     case ContextMenuAction::ContextMenuItemLastCustomTag:
     case ContextMenuAction::ContextMenuItemBaseApplicationTag:
         return true;
+    case ContextMenuAction::ContextMenuItemPDFSinglePageContinuous:
+    case ContextMenuAction::ContextMenuItemPDFTwoPages:
+    case ContextMenuAction::ContextMenuItemPDFTwoPagesContinuous:
+        return true;
```

Comment:
@sgill26 It would be good to move these up and share the `return true;` with the other cases in a follow-up patch (maybe the patch where you hook up the context menu items into PDF.js).